infra/blueprint-test/pkg/bq/bq.go

```diff
@@ -91,6 +110,11 @@ func RunCmdE(t testing.TB, cmd string, opts ...cmdOption) (string, error) {
 	if err != nil {
 		t.Fatal(err)
 	}
+
+	err = initBq()
```

Comment:
nit: Could we pass in t to initBq if we are planning to call t.Fatal(err) anyway?

Reply:
Since `initBq` doesn't currently call `t.Fatalf` inside of it (instead returning an error value), what would passing t accomplish?

Reply:
I was suggesting `initBq` could call t.Fatalf by passing `t` if we are going to call t.Fatal(err) at the call site anyway.

Reply:
Ack, fixed.